### PR TITLE
DBZ-5594 Clean up "logical name" config

### PR DESCRIPTION
**Pull request by harveyyue**
https://issues.redhat.com/browse/DBZ-5594

### CODE REVIEW

**Comment by jpechane** [comment]
@harveyyue Could you please reabse thsi PR on the latest `main`? There were refactorings related to topic prefixes. Thanks a lot

**Comment by harveyyue** [comment]
> @harveyyue Could you please reabse thsi PR on the latest `main`? There were refactorings related to topic prefixes. Thanks a lot

Done.

**Comment by jpechane** [comment]
@vjuranek Could you please review this one? It LGTM, I just want to make sure we are consistent accross all PRs.

File: debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java [hunk 1]
```diff
@@ -322,11 +320,13 @@ public static SchemaNameAdjustmentMode parse(String value) {
             .withGroup(Field.createGroupEntry(Field.Group.CONNECTION, 0))
             .withWidth(Width.MEDIUM)
             .withImportance(Importance.HIGH)
-            .withDefault(LOGIC_NAME_PLACEHOLDER)
             .withValidation(CommonConnectorConfig::validateTopicName)
             .required()
-            .withDescription("The name of the prefix to be used for all topics, the placeholder " + LOGIC_NAME_PLACEHOLDER +
-                    " can be used for referring to the connector's logical name as default value.");
+            .withDescription("Topic prefix that provides a namespace for the particular MySQL database " +
```
Comment (vjuranek) [inline debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java:325]: This is common config, so we shouldn't mention MySQL here.

**Review by vjuranek — CHANGES_REQUESTED** [review]
LGTM except small issue mentioned MySQL in common.

File: debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java (hunk 1 above)
Comment (harveyyue) [reply debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java]: @vjuranek 
Revert this config description.

File: debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java [hunk 2]
```diff
@@ -54,7 +54,6 @@
  */
 public abstract class CommonConnectorConfig {
     public static final String TASK_ID = "task.id";
-    public static final String LOGICAL_NAME = "logical.name";
     public static final String LOGIC_NAME_PLACEHOLDER = "${logical.name}";
```
Comment (vjuranek) [inline debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java:57]: `LOGIC_NAME_PLACEHOLDER` is now unused and should be removed (IIRC you had it in previous version - it was fine, I just wanted to fix the description not to refer to MySQL specifically the the config field is used also in other connectors).

**Review by vjuranek — CHANGES_REQUESTED** [review]

File: debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java (hunk 2 above)
Comment (harveyyue) [reply debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java]: done.

**Review by vjuranek — APPROVED** [review]
Thanks @harveyyue ! LGTM now.

**Comment by jpechane** [comment]
@vjuranek Will you merge and resolve the issue then?

**Comment by vjuranek** [comment]
@jpechane will do, I wasn't sure if I can merge any non-trivial changes:-)

**Comment by vjuranek** [comment]
Thanks @harveyyue , merged!

**Pull request merged**